src/main/resources/data.sql

```diff
@@ -9,16 +9,16 @@ INSERT INTO `client` (active, uid, password)
 VALUES (true, '9fbd4ae2-2db3-44c7-9544-88e80255b56e', '$2a$10$AbxhLGtIx7yv8jhF0BePiOxnb1mlHHq/Ge4R3PxCL2wIsoEov1VaS');
 
 INSERT INTO `role` (name) VALUES
-('ROLE_USER'),
-('ROLE_SUPER_USER');
+('USER'),
```

Comment:
INFO - Reverting changes?

Reply:
Yeah, only left it with default prefix to start with because I wanted to get the other changes through. This change includes blanking the prefix which is better as then users don't need to add ROLE_ to any new roles.